rs/orchestrator/src/registration.rs

```diff
@@ -222,6 +222,7 @@ impl NodeRegistration {
             // Unused section follows
             p2p_flow_endpoints: Default::default(),
             prometheus_metrics_endpoint: Default::default(),
+            node_type: None,
```

Comment:
I'm assuming this will be changed to hold an actual value once the new field is supported by the registry?

Reply:
@eichhorl Yes.  I don't know how to determine this value.  

@sasa-tomic This is what we need to have the node team populate.  I don't know where we'd find this information, but if it's readily available, this would be an easy change.